webapp/components/channel_header.jsx

```diff
@@ -759,10 +772,51 @@ export default class ChannelHeader extends React.Component {
         }
 
         let headerText;
-        if (this.state.enableFormatting) {
-            headerText = TextFormatting.formatText(channel.header, {singleline: true, mentionHighlight: false, siteURL: getSiteURL()});
-        } else {
+        let headerTextContainer;
+        if (channel.header) {
             headerText = channel.header;
```

Comment:
Is raw HTML intentionally allowed in channel headers when formatting is disabled?